_includes/content/reqs-sys.md

```diff
@@ -27,6 +27,14 @@
   > [Preventing NetworkManager from controlling {{site.prodname}} interfaces]({{ site.baseurl }}/maintenance/troubleshoot/troubleshooting#configure-networkmanager)
   > before installing {{site.prodname}}.
   {: .alert .alert-info}
+  
+- If distribution comes with installed Firewalld or another iptables manager it should be disabled. 
+  It would interfere with rules added by {{site.prodname}}/Felix and caused problems with pod communication.
```

Comment:
"These may interfere with rules added by {{site.prodname}} and result in unexpected behavior" 

I'd suggest changing the second line to this.

Reply:
resolved